src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/TypecastParenPadCheckTest.java

```diff
@@ -52,7 +54,7 @@ public void testDefault()
             "89:14: " + getCheckMessage(MSG_WS_FOLLOWED, "("),
             "89:21: " + getCheckMessage(MSG_WS_PRECEDED, ")"),
         };
-        verify(checkConfig, getPath("InputWhitespace.java"), expected);
+        verify(checkConfig, getPath("InputWhitespace[TypecastParenPad].java"), expected);
```

Comment:
Made update to original issue to explain the brackets. They were just for show and should not be in file name.

Also check name should be right after `Input` so I am expecting names like `InputTypecastParenPadWhitespace.java`, `InputTypecastParenPadAround.java`, etc.
Naming after `InputTypecastParenPad` should be similar to original name so it is easier to tell where the file came from if it was copied and not moved.